transaction/extra/Cargo.toml

```diff
@@ -20,7 +20,7 @@ zeroize = "1"
 # MobileCoin dependencies
 mc-account-keys = { path = "../../account-keys" }
 mc-crypto-dalek = { path = "../../crypto/dalek" }
-mc-crypto-digestible = { path = "../../crypto/digestible" }
+mc-crypto-digestible = { path = "../../crypto/digestible", features = ["dalek"] }
```

Comment:
Does digestible now have configurable backends?

Reply:
no -- it has an optional dependency on Dalek crates, and if the dalek feature is on, then it includes implementations of how to digest them.

if you want, we could reverse that, and dalek crate could own the implementations of digestible. it was just easier for me to do it this way because changing dalek is harder than changing our own crates

Reply:
this is what the dalek feature does: https://github.com/mobilecoinfoundation/mobilecoin/blob/master/crypto/digestible/src/lib.rs#L629

I can't remember why I turned this on, but I think some tests didn't build if I didn't have it.